text/reporting.tex

```diff
@@ -402,7 +402,17 @@ \section{Aspect 3: Subsystems Included in Instrumented Power}
 
 
 \noindent
-For Level 1, only include the compute-node subsystem. For some systems, it may be impossible not to include a power contribution from some subsystems. In this case, provide a list of the measured subsystems, but do not subtract an estimated value for the included subsystem. 
+For Level 1, the only subsystem that requires an actual power measurement is the compute-node subsystem. 
+For some systems, it may be impossible not to include a power contribution from some subsystems. 
+In this case, provide a list of the measured subsystems, but do not subtract an estimated value for the included subsystem. 
+
+For Level 1, the interconnect subsystem participating in the workload must be measured or, if not measured, the contribution 
+++must be estimated. 
++Include everything that you need to operate the network that is not part of the compute subsystem. 
++This may include infrastructure that is shared, but excludes that part that is not servicing the current cluster.
+++
+++List all other subsystems that contribute to the workload, but you do not have to provide 
```

Comment:
I would just remove this, having them list the other subsystems doesn't really help us, and is quite vague.  They could list nothing, or everything down to the cooling and lighting.